eq-author/src/components/CollapsibleToggled/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { colors } from "constants/theme";
+
+import ToggleSwitch from "components/buttons/ToggleSwitch";
+
+const Wrapper = styled.div``;
+const Header = styled.div`
+  display: flex;
+  align-items: center;
+
+  h2 {
+    font-size: 1em;
+    color: ${colors.black};
+    margin-right: 1em;
+  }
+`;
+const Body = styled.div`
+  display: ${(props) => (props.isOpen ? "block" : "none")};
+  margin-top: -1em;
+  margin-left: 0.1em;
+  padding: 0 0 0 0.5em;
+  border-left: 3px solid ${colors.lightGrey};
+`;
+
+const CollapsibleToggled = ({
+  title,
+  defaultOpen = false,
+  headerContent,
+  children,
+}) => {
+  const [isOpen, toggleCollapsible] = useState(defaultOpen);
+
+  return (
+    <Wrapper data-test="CollapsibleToggled">
+      <Header data-test="CollapsibleToggled__Header">
+        <h2>{title}</h2>
+        <ToggleSwitch
+          name={`${title}-toggle-switch`}
+          hideLabels={false}
+          onChange={() => toggleCollapsible(!isOpen)}
+          checked={isOpen}
+        />
+        {headerContent}
+      </Header>
+      <Body isOpen={isOpen} data-test="CollapsibleToggled__Body">
+        {children}
+      </Body>
```

Comment:
```suggestion
      {isOpen && <Body data-test="CollapsibleToggled__Body">{children}</Body>}
```

Not sure how you feel about this instead, it's a bit more obvious that its rendered conditionally instead of being in the css

Reply:
Sure.